src/app/genes/gene-overview/soe/soe.component.ts

```diff
@@ -15,21 +20,44 @@ export class SOEComponent implements OnInit {
     @Input() gene: Gene;
     @Input() geneInfo: GeneInfo;
     @Input() id: string;
+    @Input() geneScoresResponse: Observable<GeneScoreDistribution>;
 
     cols: any[];
     summary: any[];
+    scoresDataLoaded: boolean = false;
+    scoreCategories: string[] = [
+        'geneticsscore',
+        'literaturescore',
+        'omicsscore',
+        'logsdon',
```

Comment:
This order matters! Putting logsdon/overall score last will make it the last plot

Reply:
woohoo!

Reply:
A tiny nitpick, but can we put literature after omics? That seems to be the preferred ordering per our scientist SMEs based on relative prioritization of the different score types...